tls/s2n_connection.h

```diff
@@ -56,6 +56,12 @@ typedef enum {
 } s2n_session_ticket_status;
 
 struct s2n_connection {
+    /* The following bitfield flags are used in SAW proofs. The position of
+     * these flags is important to SAW, so make sure that any new flags are
+     * added after these ones.
```

Comment:
Could you just improve this comment with a sentence explaining why "the position of these flags is important to SAW"?